names/names.py

```diff
@@ -10,11 +11,30 @@
 names_2 = f.read().split("\n")  # List containing 10000 names
 f.close()
 
+''' 
+Orignal code needed to be replaced
+
+
 duplicates = []
 for name_1 in names_1:
     for name_2 in names_2:
         if name_1 == name_2:
             duplicates.append(name_1)
+'''
+
```

Comment:
You did a great job improving the runtime complexity of this operation. Congrats derek